src/site/_collections/hooks/utils.js

```diff
@@ -68,15 +68,20 @@ const index = (items, href, testItems) => {
 /**
  * @param {VirtualCollectionItem[]} items
  * @param {string} lang
+ * @param {boolean} indexedOnly
  * @return {Paginated[]}
  */
-const individual = (items, lang) => {
+const individual = (items, lang, indexedOnly = false) => {
   let paginated = [];
 
   for (const item of items) {
-    if (item.elements.length > 0) {
+    const elements = indexedOnly
+      ? item.elements.filter((element) => element.data.noindex !== true)
+      : item.elements;
```

Comment:
```suggestion
    let elements = item.elements;
    if (indexedOnly) {
      elements = elements.filter((element) => element.data.noindex !== true);
    }
```

With the .filter call I think not using a ternary operator here is easier to read 🙂